test/cli/conf.t

```diff
@@ -5,39 +5,39 @@ Invalid version:
 
 Exit code is printed by hand because sed succeeding would hide the error.
 
-  $ (<a.ml ocamlformat --impl -; echo [$?]) 2>&1 | sed 's/version is "[^"]*"/version is "..."/g'
-  ocamlformat: Error while parsing $TESTCASE_ROOT/.ocamlformat:
+  $ (<a.ml ocamlformat --impl -; echo [$?]) 2>&1 | sed 's/version is "[^"]*"/version is "..."/g' | make_deterministic
+  ocamlformat: Error while parsing <testcase_root>/.ocamlformat:
```

Comment:
Why is this needed ?

Reply:
Because dune doesn't pick up Windows paths and doesn't do the rewrite with TESTCASE_ROOT

Reply:
Perhaps we should disable all the tests on Windows ? We could add a windows-specific formatting test to be notified of formatting changes and that would be already better than before.

Reply:
Done, I've also added some tests for the Eol_compat lib. Everything looks good to me now.